test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/WarehouseEmployee.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@model MvcTagHelpersWebSite.Models.Warehouse
+
+@addtaghelper "Microsoft.AspNet.Mvc.TagHelpers"
+
+@{
+    ViewBag.Title = "Warehouse - Employee";
+}
+
+<html>
+<body>
+    <form asp-controller="MvcTagHelper_Home" asp-action="WarehouseEmployee" asp-anti-forgery="false">
+        <div>
+            @Html.LabelFor(m => m.City)
+            @Html.DisplayFor(m => m.City)
+        </div>
+        <div>
+            <label>@Html.DisplayNameFor(m => m.Employee.Address)</label>
+            @Html.TextAreaFor(m => m.Employee.Address);
+        </div>
+        <div>
+            @Html.LabelFor(m => m.Employee.Password)
+            @Html.PasswordFor(m => m.Employee.Password)
+        </div>
+        <div>
+            <label>@Html.DisplayNameFor(m => m.Employee.Phone)</label>
+            @Html.TextBoxFor(m => m.Employee.Phone)
+        </div>
+        <div>
+            @Html.LabelFor(m => m.Employee.Name)
+            @Html.NameFor(m => m.Employee.Name)
```

Comment:
- odd there's no input for `Name`; add something
- since you've tested `LabelFor()` above and this is an odd `NameFor()` use, suggest instead
``` html
<label for="@Html.IdFor(m => m.Employee.Name">@Html.NameFor(m => m.Employee.Name)</label>
```